trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/components/factory/VMDComponents.kt

```diff
@@ -42,20 +43,28 @@ object VMDComponents {
 
             fun withContent(
                 content: String,
+                spans: kotlin.collections.List<VMDRichTextSpan>,
```

Comment:
The param should be optional (having a default value)
+ remove the explicit package name

Reply:
Good catch. I'll add an empty list default param value. 

I used the explicit package name because `List` is declared [in the same file](https://github.com/mirego/trikot/blob/5716dd91d02e0585ff26bd3802e5a041373c7948/trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/components/factory/VMDComponents.kt#L386). The explicit package name is also in use [here](https://github.com/mirego/trikot/blob/5716dd91d02e0585ff26bd3802e5a041373c7948/trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/components/factory/VMDComponents.kt#L410).